### ISSUE TITLE
Smallest image always displayed when use min-width order in readme example config

### ISSUE THREAD

**Issue by zarocknz** [heyday/silverstripe-responsive-images#16]
Hi,

Evaluating this module to see if its suitable for a project but hit an issue which took me quite some time to figure out. Discovered that using the order of min-width as per the example config in the readme causes the smallest image to always be displayed no matter the screen size.

For example a config of...
```
ResponsiveImageExtension:
  sets:
    ResponsiveSet1:
      sizes:
        - {query: "(min-width: 1200px)", size: 800}
        - {query: "(min-width: 800px)", size: 400}
        - {query: "(min-width: 200px)", size: 100}
```
Always results in the smallest size of 100 being displayed. This lead me to believe the module was broken and I almost gave up on it.

However, discovered by trail and error that if the order of the items in the set size is reversed to...
```
ResponsiveImageExtension:
  sets:
    ResponsiveSet1:
      sizes:
        - {query: "(min-width: 200px)", size: 100}
        - {query: "(min-width: 800px)", size: 400}
        - {query: "(min-width: 1200px)", size: 800}
```
I.e the smallest size is at the top and then largest at the bottom then it works as expected and a differently sized image is displayed when the screen width is pulled in and out.

You might want to double-check and consider updating the example config in the readme.

Cheers,
DouG.

**Comment by zarocknz** [comment]
Hello again,

With the help of a front-end dev at my company we managed to figure out what is going on here.

Because there has not been a tagged release of this module for a while, our composer was installing the 0.1.1 version which is way behind the master branch and using an older version of picturefill so the order needed to be opposite of the example in the readme (as detailed above).

When we make it pull down dev-master the new version of picture fill is used and the order of the sizes then needs to be largest first as per the readme. So the readme does not need to be changed.

Can you please look to do a new tagged release of this module?

Thanks,
DouG.

**Comment by stecman** [comment]
Thanks @zarocknz, we've released the current master as `0.2.0`.
